static/app/components/replays/player/scrubberMouseTracking.tsx

```diff
@@ -0,0 +1,39 @@
+import {useCallback} from 'react';
+
+import MouseTracking, {
+  Props as MouseTrackingProps,
+} from 'sentry/components/replays/mouseTracking';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+
+type OnMouseMoveParams = Parameters<MouseTrackingProps['onMouseMove']>;
+
+type Props = {
+  children: React.ReactNode;
+};
+
+function ScrubberMouseTracking({children}: Props) {
+  const {duration = 0, setCurrentHoverTime} = useReplayContext();
+
+  const handleMouseMove = useCallback(
+    (params: OnMouseMoveParams[0]) => {
+      if (!params || duration === undefined) {
+        setCurrentHoverTime(undefined);
+        return;
+      }
+      const {left, width} = params;
+
+      if (left >= 0) {
+        const percent = left / width;
+        const time = percent * duration;
+        setCurrentHoverTime(time);
+      } else {
+        setCurrentHoverTime(undefined);
```

Comment:
Yes. Theoretically you can hover over the `0` spot, and we would want to show the line. Whereas when it's `undefined` nothing will appear in the Timeline.

but, it could also be the case that we're doing a truthy test along the way and messing it up... i'll see that we are not.

Reply:
So in https://github.com/getsentry/sentry/issues/36222 we'll make sure it's comparing to `undefined` instead of being truthy